web/app/frontend/jest.setup.js

```diff
@@ -0,0 +1,38 @@
+import 'react-native-gesture-handler/jestSetup';
+import mockAsyncStorage from '@react-native-async-storage/async-storage/jest/async-storage-mock';
+import 'core-js-api-client';
+
+jest.mock('@react-native-async-storage/async-storage', () => mockAsyncStorage);
```

Comment:
is there a reason there are two files with this? or can one of them be removed?

Reply:
not sure what you mean

Reply:
I mean this file: web/app/frontend/__mocks__/@react-native-async-storage/async-storage.js
it basically has this exact line as content, and I'm assuming one can go?